src/components/Owner/OwnerProfile.jsx

```diff
@@ -50,31 +56,33 @@ const OwnerProfile = ({ setViewValue, barId, count }) => {
         setOpen(false);
     };
 
-    const occLevel = () => {
-        count
-    }
 
-    const [barAddress, setBarAddress] = useState('');
-    const [barNumber, setBarNumber] = useState('');
-    const [barName, setBarName] = useState('')
-    const [image, setImage] = useState('');
-    const [occ, setOcc] = useState('');
-
-    const barInfo = () => {
-        axios.get(`/db/bar/info?id=${barId}`)
-            .then(({ data }) => {
-                setImage(data[0].profile_image);
-                setBarName(data[0].bar_name);
-                setBarAddress(data[0].address)
-                setBarNumber(data[0].phone_number)
-                setOcc(data[0].capacity);
-            })
+    let covidCap = (capacity * .25)
+    let covidCapLow = covidCap * .25;
+    let covidCapHigh = covidCap * .75
+    const occupencyStatus = () => {
+        if (count < covidCapLow) {
+            console.log(count, (covidCap * .25), 'COUNT')
```

Comment:
take out console.log. Your linter/husky should not be allowing this push to go through with all of these console.logs in